ContuityTests/ModelsTests/JotTests.swift

```diff
@@ -28,4 +33,44 @@ class JotTests: XCTestCase {
         XCTAssertEqual(statement?.columnCount, 7)
         XCTAssertEqual(statement?.columnNames, ["id", "data", "queue", "createdAt", "modifiedAt", "latitude", "longitude"])
     }
+    
+    func testRead() {
```

Comment:
add a comment explaining whats being tested

Reply:
to add to the above, another alternative would be make function headers like `testCallToXDoesYAndReturnsZ()` which is like some of the other tests